src/ig/IGExporter.ts

```diff
@@ -1143,6 +1150,32 @@ export class IGExporter {
     }
   }
 
+  private getMetaExtensionDescription(resource: InstanceDefinition): string {
+    const description = resource.meta?.extension?.find(
+      e => e.url === 'http://hl7.org/fhir/StructureDefinition/instance-description'
+    )?.valueString;
+    const definition = this.fhirDefs.fishForFHIR(resource.resourceType, Type.Resource);
+    if (
+      definition &&
+      !definition.snapshot.element.find((e: any) => e.id === `${resource.resourceType}.description`)
```

Comment:
I feel like this is more aggressive than it should be.  It may capture some non-conformance resources that happen to have fields named description or title.  For example [Goal](http://hl7.org/fhir/R4/goal.html) has a description field, but authors may still want to put a description for the example (notably because `Goal.description` is a `CodeableConcept`.  I think you should only exclude the "conformance" and "terminology" resource types, as described in the Jira issue.

I know Grahame said "they won't be allowed on resources that have title and description elements", but I think he just meant "conformance" and "terminology" resources.  But if you really want to implement what he said literally then you should check for both "title" and "description" being present at the same time.

Reply:
Yeah I misread the Jira issue and used the constraints that they voted out. Fixed in https://github.com/FHIR/sushi/pull/666/commits/8eeb5f2771d24aec630231b19767d237744ba9f9.

Reply:
Actually, I missed that part in the Jira issue, so now I'm less sure.  It *does* kind of look like they intended it to be the way you originally coded it, but as I noted, that has flaws.  For now, I like the way you fixed it better anyway, so let's keep it like that until we hear otherwise.

Reply:
Unless I'm misreading the issue again they added this:
```
`%resource.title.exists().not()` for the name one and `%resource.description.exists().not()` for the description one
```
as a description on the 2020-11-02, but then changed that to this:
```
Will also define appearance constraints to ensure that these extensions aren't used in resources where there is a "proper" (non-extension-based) slot for name/description (i.e., conformance resources cannot use these extensions). (FHIRPath reflection allows for this kind of constraint, saying the resource type does not have an element where name = 'title', see http://hl7.org/fhirpath/#reflection)
```
Three days ago. So I think the way it is now is more consistent with the current description.